flexget/plugins/internal/change_warn.py

```diff
@@ -31,6 +31,10 @@ def on_task_start(self, task, config):
             log.critical('Plugin nzb_size is deprecated, use content_size instead')
             found_deprecated = True
 
+        if 'disable_urlrewriters' in task.config:
+            log.critical('Plugin disable_urlrewriters is deprecated, use disable instead')
```

Comment:
Wouldn't a warning fit here better? Also, why disable the old way right away? You can set a depreciation message on that plugin for now.

Reply:
Well, this change warn was silly turns out. We won't ever get here if I leave it removed (because schema validation will fail.) The main reason I wanted to delete it right now is because we don't have a good system to follow up on deleting deprecated things.

Reply:
Re-added this plugin which just calls the disable plugin methods now and has a deprecation warning.